docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -269,10 +265,11 @@ public Node call() throws Exception {
                                     //
                                     // deferring the completion of provisioning until the launch
```

Comment:
Sorry, but when launcher will be used then? And it seems that in case of jnlp this may look crazy, though i can have some timeout and do connection retries from docker side. 
So backing to my initial idea, i can move current launcher configuration to separate launcher class (unbundle hardcodes), then i can have some (static) methods that i can call from Provisioner for determining that connection is ok, like i did method for configuring dockerCreate parameters.  And then Cloud API will use "standard" launcher methods?